experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -77,6 +77,24 @@ class Flowables {
         std::forward<OnSubscribe>(function)));
   }
 
+  template <typename T>
+  static Reference<Flowable<T>> empty() {
+    auto lambda = [](Subscriber<T>& subscriber, int64_t) {
+      subscriber.onComplete();
+      return std::make_tuple(static_cast<int64_t>(0), true);
+    };
+    return Flowable<T>::create(std::move(lambda));
+  }
+
+  template <typename T>
+  static Reference<Flowable<T>> error(const std::exception_ptr ex) {
```

Comment:
I wonder if you'll also want an overload `Flowables::error(std::exception)`.  Might be a nice wrapper for code like:

`Flowables::error<int>(std::make_exception_ptr(std::runtime_error("oops")))`